azure-pipelines.yml

```diff
@@ -76,7 +76,14 @@ stages:
             filePath: ./eng/install-procdump.ps1
             failOnStderr: true
             showWarnings: true
-
+        - task: PowerShell@2
```

Comment:
Note: You can try to borrow from https://github.com/CommunityToolkit/Windows/blob/e06140d949748cdf79e327ce7f244fc311fe26cd/.github/workflows/build.yml#L85-L92

Reply:
isn't this the same code?

Reply:
@nohwnd It's slightly different and I thought that's why it fails.

You do:

```
New-Item -Path "HKLM:\SOFTWARE\Microsoft\Windows\Windows Error Reporting" -Name "LocalDumps"
```

But looks like this already exists and crashes. The other samples I linked create the properties right away.

I think it's mostly that `New-Item -Path "HKLM:\SOFTWARE\Microsoft\Windows\Windows Error Reporting" -Name "LocalDumps"` needs to be removed?